File-Formats/DocIO/convert-markdown-to-word-document-in-csharp.md

```diff
@@ -0,0 +1,575 @@
+//Open the file as a Stream.
+using (Stream docStream = typeof(App).GetTypeInfo().Assembly.GetManifestResourceStream("Sample.Assets.Input.md"))
+{
+    //Load the file stream into a Markdown file.
+    using (WordDocument document = new WordDocument(docStream, FormatType.Markdown))
+    {
+        //Save as a Word document into the MemoryStream.
+        MemoryStream stream = new MemoryStream();
+        await document.SaveAsync(stream, FormatType.Docx);
+        //Save the stream as a Word document in the local machine.
+        Save(stream, "MarkdownToWord.docx");
+    }
+//Please refer to the following link to save the Word document in UWP platform
+//https://help.syncfusion.com/file-formats/docio/create-word-document-in-uwp#save-word-document-in-uwp
+}
+{% endhighlight %}
+
+{% highlight c# tabtitle="ASP.NET Core" %}
+//Open the file as a Stream.
+using (FileStream docStream = new FileStream("Input.md", FileMode.Open, FileAccess.Read))
+{
+    //Load the file stream into a Markdown file.
+    using (WordDocument document = new WordDocument(docStream, FormatType.Markdown))
+    {
+        //Save as a Word document into the MemoryStream.
+        MemoryStream outputStream = new MemoryStream();
+        document.Save(outputStream, FormatType.Docx);
+        outputStream.Position = 0;
+        //Download the Word document in the browser.
+        return File(outputStream, "application/msword", "MarkdownToWord.docx");
+    } 
+}
+{% endhighlight %}
+
+{% highlight c# tabtitle="Xamarin" %}
+//Open the file as a Stream.
+using (Stream docStream = typeof(App).GetTypeInfo().Assembly.GetManifestResourceStream("Sample.Assets.Input.md"))
+{
+    //Load the file stream into a Markdown file.
+    using (WordDocument document = new WordDocument(docStream, FormatType.Markdown))
+    {
+        //Save as a Word document into the MemoryStream.
+        MemoryStream outputStream = new MemoryStream();
+        document.Save(outputStream, FormatType.Docx);
+        //Save the stream as a file in the device and invoke it for viewing. 
+        Xamarin.Forms.DependencyService.Get<ISave>().SaveAndView("MarkdownToWord.docx", "application/msword", outputStream);
+    }
+//Please download the helper files from the following link to save the stream as a file and open the file for viewing in the Xamarin platform.
+//https://help.syncfusion.com/file-formats/docio/create-word-document-in-xamarin#helper-files-for-xamarin
+}
+{% endhighlight %}
+{% endtabs %}
+
+T> You can also save the markdown file as [HTML](https://help.syncfusion.com/file-formats/docio/html), [PDF](https://help.syncfusion.com/file-formats/docio/word-to-pdf) and [Image](https://help.syncfusion.com/file-formats/docio/word-to-image).
+
+## Customize image data
+
+The .NET Word (DocIO) library provides an MdImageNodeVisited event, which is used to customize image data while importing Markdown file. You can implement the logic to customize the image data by using this MdImageNodeVisited event.
```

Comment:
The .NET Word (DocIO) library provides a MdImageNodeVisited event, which customizes image data while importing a Markdown file. Implement the logic to customize the image data by using this MdImageNodeVisited event.